src/main/kotlin/rys/ajaxpetproject/service/impl/MessageServiceImpl.kt

```diff
@@ -0,0 +1,41 @@
+package rys.ajaxpetproject.service.impl
+
+import org.bson.types.ObjectId
+import org.springframework.stereotype.Service
+import rys.ajaxpetproject.exception.MessageNotFoundException
+import rys.ajaxpetproject.exception.MessagesFromChatNotFoundException
+import rys.ajaxpetproject.model.MongoMessage
+import rys.ajaxpetproject.repository.MessageRepository
+import rys.ajaxpetproject.service.MessageService
+
+@Service
+class MessageServiceImpl(val messageRepository: MessageRepository, val chatService: ChatServiceImpl) : MessageService {
```

Comment:
what is purpose of ChatService interface if you use in other classes direct implementation?

Reply:
Didn't saw, my bad